gtk3/sample/misc/aboutdialog2.rb

```diff
@@ -2,7 +2,7 @@
 =begin
   aboutdialog2.rb - Ruby/GTK sample script.
 
-  Copyright (c) 2005-2015 Ruby-GNOME2 Project Team
+  Copyright (c) 2006-2015 Ruby-GNOME2 Project Team
```

Comment:
The original started at 2006 . The last commit is a fix to my previous error on this copyright.